### ISSUE TITLE
lnd not reachable

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud-docker#93]
```
xucli getinfo
      "LTC",
      {
        "error": "lnd is Disconnected",
```
```
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
```

```
testnet_lndbtc_1      ./entrypoint.sh --alias=ln ...   Up      0.0.0.0:10009->10009/tcp, 0.0.0.0:9735->9735/tcp                         
testnet_lndltc_1      ./entrypoint.sh --alias=ln ...   Up      0.0.0.0:20009->10009/tcp, 0.0.0.0:10735->9735/tcp 
```

**Comment by kilrau** [comment]
Saw this again today, first working, then not reachable:
```
testnet > getinfo
D0723 11:26:39.567458535      48 env_linux.cc:71]            Warning: insecure environment read function 'getenv' used
{
  "version": "1.0.0-testnet.1",
  "nodePubKey": "030c74ebef2068dfa48c8cae1a33b670528cf10d636d11f8f9ee61e89f02aba93e",
  "urisList": [
    "030c74ebef2068dfa48c8cae1a33b670528cf10d636d11f8f9ee61e89f02aba93e@0.0.0.0:8885"
  ],
  "numPeers": 0,
  "numPairs": 1,
  "orders": {
    "peer": 0,
    "own": 0
  },
  "lndMap": [
    [
      "BTC",
      {
        "error": "",
        "channels": {
          "active": 0,
          "inactive": 0,
          "pending": 0
        },
        "chainsList": [
          {
            "chain": "bitcoin",
            "network": "testnet"
          }
        ],
        "blockheight": 1570525,
        "urisList": [],
        "version": "0.6.1-beta commit=v0.6.1-beta",
        "alias": "lndbtc"
      }
    ],
    [
      "LTC",
      {
        "error": "lnd is OutOfSync",
        "chainsList": [],
        "blockheight": 0,
        "urisList": [],
        "version": "",
        "alias": ""
      }
    ]
  ],
  "raiden": {
    "error": "",
    "address": "0xa65dda2A2889263F6E254d1E91E1F7c44211CFD2",
    "channels": 0,
    "version": ""
  }
}
testnet > lndltc-lncli getinfo
{
	"version": "0.6.1-beta commit=v0.6.1-beta",
	"identity_pubkey": "02d4985109c24b73cf21b3c5e1527393d720261e28c641efb6a5e7fcc6f012336c",
	"alias": "lndltc",
	"num_pending_channels": 0,
	"num_active_channels": 0,
	"num_inactive_channels": 0,
	"num_peers": 0,
	"block_height": 1153954,
	"block_hash": "ccb002f67d33ddfd063134ae715a2105abda482922cc74c820d3d862eb3cdc9a",
	"best_header_timestamp": 1562929051,
	"synced_to_chain": false,
	"testnet": true,
	"chains": [
		{
			"chain": "litecoin",
			"network": "testnet"
		}
	],
	"uris": null
}
testnet > restart lndltc
Restarting testnet_lndltc_1 ... done
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
```

**Comment by kilrau** [comment]
Actually it looks like `lndltc` is restarting, even though docker doesn't show that:
```
testnet > docker-compose ps
       Name                      Command               State                                     Ports                                  
----------------------------------------------------------------------------------------------------------------------------------------
testnet_bitcoind_1    bitcoind -server -testnet  ...   Up      0.0.0.0:18332->18332/tcp, 0.0.0.0:28332->28332/tcp,                      
                                                               0.0.0.0:28333->28333/tcp, 8332/tcp                                       
testnet_litecoind_1   ./entrypoint.sh -server -t ...   Up      0.0.0.0:19332->19332/tcp, 0.0.0.0:29332->29332/tcp,                      
                                                               0.0.0.0:29333->29333/tcp, 9332/tcp                                       
testnet_lndbtc_1      ./entrypoint.sh --alias=ln ...   Up      0.0.0.0:10009->10009/tcp, 0.0.0.0:9735->9735/tcp                         
testnet_lndltc_1      ./entrypoint.sh --alias=ln ...   Up      0.0.0.0:20009->10009/tcp, 0.0.0.0:10735->9735/tcp                        
testnet_parity_1      ./entrypoint.sh --no-warp  ...   Up      30303/tcp, 0.0.0.0:30303->30303/udp, 5001/tcp, 8080/tcp, 8082/tcp,       
                                                               8083/tcp, 8180/tcp, 0.0.0.0:8545->8545/tcp, 0.0.0.0:8546->8546/tcp       
testnet_raiden_1      ./entrypoint.sh --keystore ...   Up      0.0.0.0:5001->5001/tcp                                                   
testnet_xud_1         /bin/sh -c ./entrypoint.sh       Up      0.0.0.0:8885->8885/tcp, 0.0.0.0:8886->8886/tcp, 0.0.0.0:8887->8887/tcp   
testnet > lndltc-lncli getinfo
[lncli] Wallet is encrypted. Please unlock using 'lncli unlock', or set password using 'lncli create' if this is the first time starting lnd.
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
testnet > lndltc-lncli getinfo
[lncli] Wallet is encrypted. Please unlock using 'lncli unlock', or set password using 'lncli create' if this is the first time starting lnd.
testnet > lndltc-lncli getinfo
[lncli] Wallet is encrypted. Please unlock using 'lncli unlock', or set password using 'lncli create' if this is the first time starting lnd.
testnet > lndltc-lncli getinfo
[lncli] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:10009: connect: connection refused"
```

**Comment by kilrau** [comment]
Note docker / docker-compose does NOT show the lnd container as "restarting", always as "up".

**Comment by kilrau** [comment]
Note: deleting the lndltc data directory did not help

**Comment by kilrau** [comment]
It has to do with the connection to litecoind/bitcoind. Restarted litecoind:
```
testnet > restart litecoind
Restarting testnet_litecoind_1 ... done
```
and lndltc is responding again:
```
testnet > lndltc-lncli getinfo
{
	"version": "0.6.1-beta commit=v0.6.1-beta",
	"identity_pubkey": "02792f6ce68d69f80b39764a2b5caee43f7d97440d5051c0b60375460654214c7a",
	"alias": "lndltc",
	"num_pending_channels": 0,
	"num_active_channels": 0,
	"num_inactive_channels": 0,
	"num_peers": 0,
	"block_height": 789429,
	"block_hash": "6c814f2fff9721aa5e938cd61c08f4d881c8377d7a59c9dccedc26288972aecc",
	"best_header_timestamp": 1486964776,
	"synced_to_chain": false,
	"testnet": true,
	"chains": [
		{
			"chain": "litecoin",
			"network": "testnet"
		}
	],
	"uris": null
}
```

**Comment by kilrau** [comment]
Seems fixed with 0.7.1
